core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -72,8 +73,8 @@ case class EntityAndAttributesResult(entityRecord: EntityRecord,
 )
 
 object EntityComponent {
-  // the length of the all_attribute_values column, which is TEXT, -1 becaue i'm nervous
-  val allAttributeValuesColumnSize = 65534
+  // the length of the all_attribute_values column, which is TEXT, minus a few bytes because i'm nervous
+  val allAttributeValuesColumnSize = 65532
```

Comment:
65532 is divisible by 4 so it makes the unit test work better, and only drops 2 bytes.